### PR TITLE
html update

### PR DESCRIPTION
**Pull request by arpita-sahakar**

### CODE REVIEW

File: registration1.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                <tr>
+                    <td><label for="date">Date: </label></td>
+                    <td>  <input type="date" name="date" id="date"></td>
+                </tr>
+
+                <tr>
+                    <td><label for="fname">Phone No:</label></td> 
+                    <td><input type="text" id="fname" name="fname"><br><br></td> 
+                </tr>
+
```
Comment (bhopii) [inline registration1.html:55]: Please remove extra lines

File: registration1.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+                                checked>
+                            <label for="scales">Hindi</label></td>
+                    
+                        <td><input type="checkbox" id="horns" name="horns">
+                            <label for="horns">English</label></td>
+                </tr>
+            
+            <tr>
+                <td>
+                 <label for="pet-select">Choose a Course:</label>
```
Comment (bhopii) [inline registration1.html]: please use meaningful variable names

**Review by bhopii — CHANGES_REQUESTED** [review]
Please work the review comments

File: registration1.html (hunk 1 above)
Comment (arpita-sahakar) [reply registration1.html:55]: ok will take care

File: registration1.html (hunk 2 above)
Comment (arpita-sahakar) [reply registration1.html]: changes done

**Review by bhopii — APPROVED** [review]
LGTM

**Pull request merged**